### PR TITLE
change top bar logic

### PR DESCRIPTION
**Pull request by klokidis**
bc the top bar is not inside the NavHost it doesn't have the tween(50) and the title changes faster than the screen so it looks a little off

### CODE REVIEW

File: app/src/main/java/com/mensinator/app/navigation/MensinatorApp.kt [hunk 1]
```diff
@@ -145,23 +144,45 @@ fun MensinatorApp(
             navController = navController,
             startDestination = Screen.Calendar.name,
             modifier = Modifier.padding(paddingValues),
-            enterTransition = { fadeIn(animationSpec = tween(50)) },
+            enterTransition = { fadeIn(animationSpec = tween(5000)) },
             exitTransition = { fadeOut(animationSpec = tween(50)) },
         ) {//create a new file for every page and pass it inside the composable
             composable(route = Screen.Calendar.name) {
-                CalendarScreen()
+                Scaffold(
```
Comment (carstenhag) [inline app/src/main/java/com/mensinator/app/navigation/MensinatorApp.kt:151]: Not sure whether this is correct, with this there are now two nested Scaffolds?

Comment (klokidis) [reply app/src/main/java/com/mensinator/app/navigation/MensinatorApp.kt:151]: yeah it works well but yeah its too a bit much. the perfect for me would be not to have a scaffold with a topbar but just a title in every screen. but we cant have the topbar change outside the nav animation

Comment (EmmaTellblom) [reply app/src/main/java/com/mensinator/app/navigation/MensinatorApp.kt:151]: Should we merge or look more at this?

Comment (klokidis) [reply app/src/main/java/com/mensinator/app/navigation/MensinatorApp.kt:151]: Nested scarfolds is possible and correct. We can accept this or we can change the scaffold topbar to just a title in every screen

**Review by EmmaTellblom — APPROVED** [review]

**Comment by carstenhag** [comment]
@klokidis with `   enterTransition = { fadeIn(animationSpec = tween(5000)) },` the screen animations look suuuper slow, I will revert that line 😄

**Comment by klokidis** [comment]
> @klokidis with ` enterTransition = { fadeIn(animationSpec = tween(5000)) },` the screen animations look suuuper slow, I will revert that line 😄

oops that was a mistake :P

**Pull request merged**